lambda_species/lambda_function.py

```diff
@@ -0,0 +1,48 @@
+import json
+import boto3
+
+def lambda_handler(event, context):
+	# Create a connection to the DynamoDB instance of AWS
+	dynamodb = boto3.resource('dynamodb')
+	
+	# Create a connection to the specific table we want from our DynamoDB. In this case, 'CrowFacts'.
```

Comment:
Change comment to match what's in the code
```suggestion
	# Create a connection to the specific table we want from our DynamoDB. In this case, 'CrowFactsTwo'.
```

Reply:
Yes TY, copy pasta strikes again